ec2.tf

```diff
@@ -0,0 +1,160 @@
+# Security group
+resource "aws_security_group" "this" {
+  count       = var.create_security_group && var.create_ec2_instance ? 1 : 0
```

Comment:
I have tested the module and it works well as defined. However I have a question
- Are there instances/cases where we would not use a security group with ECS EC2 instances?

Reply:
Uhhm no, but if you don't create your own security group, a new one(default) will automatically be created for you.